pytorch_lightning/callbacks/finetuning.py

```diff
@@ -136,16 +198,22 @@ def on_before_accelerator_backend_setup(self, _, pl_module):
     def on_train_epoch_start(self, trainer, pl_module):
         """Called when the epoch begins."""
         for opt_idx, optimizer in trainer.train_loop.prepare_optimizers():
-            self.finetunning_function(pl_module, trainer.current_epoch, optimizer, opt_idx)
+            self.finetune_function(pl_module, trainer.current_epoch, optimizer, opt_idx)
 
-    def finetunning_function(self, pl_module: LightningModule, epoch: int, optimizer: Optimizer, opt_idx: int):
+    def finetune_function(self, pl_module: LightningModule, epoch: int, optimizer: Optimizer, opt_idx: int):
+        """
+        Override to add your unfreeze logic
+        """
         raise NotImplementedError
 
     def freeze_before_training(self, pl_module: LightningModule):
+        """
+        Override to add your freeze logic
+        """
         raise NotImplementedError
 
 
-class BackboneLambdaFinetuningCallback(BaseFinetuningCallback):
+class BackboneFinetuning(BaseFinetuning):
```

Comment:
Comments to the arg description of this function:
* `verbose` and `should_align` is listed twice
* they should be ordered similar to argument order in the `__init__` function

Reply:
Good catch !